packages/dds/merge-tree/src/referencePositions.ts

```diff
@@ -3,7 +3,8 @@
  * Licensed under the MIT License.
  */
 
-// eslint-disable-next-line import/no-deprecated
+/* eslint-disable import/no-deprecated */
```

Comment:
Nit: disabling this at the file level will prevent us from catching unexpected usages of other deprecated items. As cumbersome as it is, I think we need to disable this rule on a line-by-line basis.

Reply:
I feel decently strongly that a file-wide ignore is the better option here. This disable will be immediately removed in the PR targeting next, and so will be short-lived in main (<2 weeks) and not exist at all in next. Additionally, any PR merging into main would run into this lint during the merge into next. There are >50 removed types and functions in the upcoming release, and adding an eslint ignore at every usage site clutters the code and causes merge conflicts -- unnecessary work in the name of solving an issue that doesn't exist, though I agree in the general case that file-wide ignores are bad.